src/scripts/achievements/AchievementHandler.ts

```diff
@@ -20,7 +20,7 @@ class AchievementHandler {
     }
 
     public static calculateNumberOfTabs() {
-        this.numberOfTabs(Math.floor((this.achievementListFiltered().filter(a => a.region <= player.highestRegion()).length - 1) / 10));
+        this.numberOfTabs(Math.max(1, Math.floor(this.achievementListFiltered().filter(a => a.region <= player.highestRegion()).length / 10)));
```

Comment:
I _think_ this solves the 1/2 pages when filtered for no achievements.
```suggestion
        this.numberOfTabs(Math.max(0, Math.floor(this.achievementListFiltered().filter(a => a.region <= player.highestRegion()).length / 10)));
```